website/docs/tutorials/06-deploying/02-migrations/call.md

```diff
@@ -0,0 +1,50 @@
+# `call`
+
+```python
+def call(
+    contract_address: int,
+    function_name: str,
+    inputs: Optional[Union[List[int], Dict[str, Any]]] = None,
+) -> NamedTuple:
+    ...
+```
+Calls a StarkNet contract without affecting the state.
```

Comment:
whose state is not affected? Mine or called contract's?

Reply:
Starknet state I guess. I copied this sentence from Starknet's docs.

https://docs.starknet.io/docs/CLI/commands/#starknet-call

Reply:
OK. But I think it would still be nice to clarify this